Content.Server/StationEvents/Events/BluespaceLockerRule.cs

```diff
@@ -45,6 +46,8 @@ protected override void Started(EntityUid uid, BluespaceLockerRuleComponent comp
             _bluespaceLocker.GetTarget(potentialLink, comp, true);
             _bluespaceLocker.BluespaceEffect(potentialLink, comp, comp, true);
 
+            EnsureComp<ArrivalsBlacklistComponent>(potentialLink); // To stop people getting to arrivals terminal
```

Comment:
Wouldn't this only blacklist one of the two lockers since the other one is assigned in `GetTarget`?
This should probably be moved to the `OnStartup` function for the `BluespaceLockerComponent` so that all bluespace lockers are arrival shuttle immune.

Reply:
You were right there, good catch. One though I had for not putting it in `OnStartup` was that to allow a bluespace locker on arrivals, you'd have to manually remove the component, but I now reaslise that's not really an issue.
![image](https://github.com/user-attachments/assets/5e2cf0be-576a-47bb-b558-8a85f0386f1f)
![image](https://github.com/user-attachments/assets/c894a750-6864-42f0-928a-225203a3f92e)

Reply:
> This should probably be moved to the OnStartup function for the BluespaceLockerComponent so that all bluespace lockers are arrival shuttle immune.

Yeah so now tests fail because it gains a component on spawn.